src/main/java/pl/gov/coi/pomocua/ads/materialaid/domain/dto/MaterialAidOfferDto.java

```diff
@@ -0,0 +1,31 @@
+package pl.gov.coi.pomocua.ads.materialaid.domain.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import pl.gov.coi.pomocua.ads.dictionaries.domain.City;
+
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.io.Serial;
+import java.io.Serializable;
+
+@Setter
+@Getter
+@Builder
+public class MaterialAidOfferDto implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 4899363368417701147L;
+    private static final String EXCLUDING_SPECIAL_CHARS = "^[^<>()%#@”‘]*$";
```

Comment:
I know these chars where specified in the docs, but I don't see why we should be worried about those chars:
* % - there is no sql injection risk, as we're not building queries by contatenation
* <> - Angular will escape those, so XSS risk in handled

This rule is way too strict. These are chars that should be allowed in the description/content.

Reply:
Discuss it with the business side then ;)